### ISSUE TITLE
Allow empty glimmer component class with decorator

### ISSUE THREAD

**Issue by adrigzr** [ember-cli/eslint-plugin-ember#1373]
It should be allowed to add an empty glimmer component class with a decorator that could provide some functionality.

Example:

```js
import Component from '@glimmer/component';
import MyDecorator from 'my-decorator';

@MyDecorator
class MyComponent extends Component {}
```
